apiserver/pkg/util/config.go

```diff
@@ -31,4 +31,10 @@ const (
 
 	// The component name for apiserver
 	ComponentName = "kuberay-apiserver"
+
+	RayServiceOriginatedFromLabelValueType = "rayservice"
 )
+
+func RayOriginatedFromServiceLabelValue(name string) string {
```

Comment:
Is it possible for the API server to use the function under `ray-operator/controllers/ray/utils` directly?

Reply:
Yes, it is. Actually, there are also several other constants duplicated from the `ray-operator/controllers/ray/utils`. We could consider reusing them in the follow-up RPs.